semcore/popper/src/Popper.jsx

```diff
@@ -370,15 +370,15 @@ class Popper extends Component {
   };
 
   getTriggerProps() {
-    const { visible, interaction, disableEnforceFocus } = this.asProps;
+    const { visible, interaction, disableEnforceFocus, explicitTriggerSet } = this.asProps;
     // @ts-ignore
     const { onKeyDown, ...interactionProps } = this.handlersFromInteraction(
       interaction,
       'trigger',
       visible,
     );
     return {
-      ref: this.createTriggerRef,
+      ref: explicitTriggerSet ? undefined : this.createTriggerRef,
```

Comment:
What do you think, if we will use `forkRef(this.asProps.forwardRef, this.createTriggerRef)` ?

And in Tooltip in d3-charts we will create ref and add `getBoundaryRect` method for them